src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-coe.mdx

```diff
@@ -18,18 +18,27 @@ redirects:
   - /telemetry-data-platform/get-started/manage-data/data-governance
 ---
 
+import rolespracticesicon from './images/oma-oe-dg-roles-practices-icon.png'
```

Comment:
these may need to be separated by empty lines above and below

Reply:
@roadlittledawn, thanks for the line spacing tip!

Reply:
@roadlittledawn OK, I'll add line feeds.